### DISCUSSION TITLE
Can't launch git installer

### DISCUSSION THREAD

**Discussion by ygolan93** [community/community#138702] [New to GitHub]
I Can't launch the program, not even with administrator access. When I click it nothing happens, doesn't even show anything on the task manager. It's the Git-2.46.0-64-bit version. Would love to have some help here 🙏

**Comment by davevad93** [comment]
Hi @ygolan93 ,

Have you tried launching it from the command line? Go to the folder where the git installer is located, open `cmd` and run the following command:

```cmd
start Git-2.46.0-64-bit.exe
```

**Comment by ygolan93** [comment]
Heleluja 🙏  it worked, tysm!

**Reply by davevad93** [reply]
You're welcome @ygolan93 👍

**Comment by indrajithbandara** [comment]
Problem sloved?

**Reply by ygolan93** [reply]
It is :)
